### DISCUSSION TITLE
Confirmation code

### DISCUSSION THREAD

**Discussion by iansquenet** [Dineshkarthik/telegram_media_downloader#465] [Q&A] [answered]
It seems a little sketchy having to give a confirmation code, how can I be sure that no one can get into my account?

**Comment by Dineshkarthik** [comment, marked as answer]
If you don't know about how MFA works and how almost all the API for any application or website uses one or other form of token just go and google and read about it. Read telegrams official API documentation instead saying it is _sketchy_ without having any knowledge of how things works.
